backend/ops_api/ops/urls.py

```diff
@@ -172,7 +172,7 @@ def register_api(api_bp: Blueprint) -> None:
     )
 
     api_bp.add_url_rule(
-        "/can-funding-summary/<int:id>",
```

Comment:
The endpoints for `*ITEM_API_VIEW*` are designed to take an resource id, e.g. `<int:id>` while the `*LIST_API_VIEW` does not.  It would be more consistent if `/can-funding-summary/` used `CAN_FUNDING_RECEIVED_LIST_API_VIEW_FUNC`.